src/App/PropertyUnits.cpp

```diff
@@ -357,6 +357,79 @@ PropertyStiffness::PropertyStiffness()
     setUnit(Base::Unit::Stiffness);
 }
 
+//**************************************************************************
+//**************************************************************************
+// PropertyMass
+//++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+TYPESYSTEM_SOURCE(App::PropertyMass, App::PropertyQuantity)
+
+PropertyMass::PropertyMass()
+{
+    setUnit(Base::Unit::Mass);
+}
+
+//**************************************************************************
+//**************************************************************************
+// PropertyAngleOfFriction
+//++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+TYPESYSTEM_SOURCE(App::PropertyAngleOfFriction, App::PropertyQuantityConstraint)
+
+PropertyAngleOfFriction::PropertyAngleOfFriction()
+{
+    setUnit(Base::Unit::AngleOfFriction);
+    setConstraints(&AngleStandard);
+}
+
+//**************************************************************************
+//**************************************************************************
+// PropertyDensity
+//++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+TYPESYSTEM_SOURCE(App::PropertyDensity, App::PropertyQuantity)
+
+PropertyDensity::PropertyDensity()
+{
+    setUnit(Base::Unit::Density);
+}
+
+//**************************************************************************
+//**************************************************************************
+// PropertyTimeSpan

```

Comment:
Here I would use a generic property "Time". This can then be used for all kinds of times with the SI unit s.

Reply:
OK, changed.